### PR TITLE
[protocol-utils] tslint->eslint

### PR DESCRIPTION
**Pull request by eobbad**
## Checklist:

-   [x] Prefix PR title with `[WIP]` if necessary.
-   [x] Add tests to cover changes as needed.
-   [x] Update documentation as needed.
-   [x] Add new entries to the relevant CHANGELOG.jsons.

### CODE REVIEW

File: packages/protocol-utils/package.json [hunk 1]
```diff
@@ -19,8 +19,8 @@
         "test:coverage": "mkdir -p ./coverage && nyc npm run test --all && yarn coverage:report:lcov",
         "coverage:report:lcov": "nyc report --reporter=text-lcov > coverage/lcov.info",
         "clean": "shx rm -rf lib generated_docs",
-        "lint": "tslint --format stylish --project .",
-        "fix": "tslint --fix --format stylish --project .",
+        "lint": "eslint src --max-warnings 256",
```
Comment (kyu-c) [inline packages/protocol-utils/package.json:22]: nit: maybe lower this unless the current number of warnings is 256?

File: packages/protocol-utils/package.json [hunk 2]
```diff
@@ -43,22 +43,24 @@
...
+        "eslint-config-prettier": "^8.5.0",
         "ethereum-types": "^3.7.1",
         "mocha": "^6.2.0",
         "npm-run-all": "^4.1.2",
         "shx": "^0.2.2",
         "sinon": "^4.0.0",
-        "tslint": "5.11.0",
         "typedoc": "~0.16.11",
-        "typescript": "4.6.3",
+        "typescript": "*",
```
Comment (kyu-c) [inline packages/protocol-utils/package.json:63]: unintended?

Comment (eobbad) [reply packages/protocol-utils/package.json:63]: happened automatically /:

File: packages/protocol-utils/CHANGELOG.json [hunk 3]
```diff
@@ -1,4 +1,13 @@
 [
+    {
+        "version": "11.17.00",
```
Comment (kyu-c) [inline packages/protocol-utils/CHANGELOG.json:3]: nit: Since it's an internal changes `11.16.12`?

**Review by kyu-c — APPROVED** [review]
LGTM

**Pull request merged**